sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_models.py

```diff
@@ -391,6 +391,15 @@ def __init__(self, **kwargs):
         self.error = kwargs.get("error", None)
         self.is_error = True
 
+    def __getattr__(self, attr):
+        if attr not in ["id", "error", "is_error"]:
```

Comment:
What are you trying to express here? Since you have an 'id', 'error' and 'is_error' attribute on the instance (unless someone has gone out of their way to delete it - which would be odd indeed), `__getattr__` won't be called for those attributes. 

On the other hand, if someone would call, let's say, `DocumentError().smurf`, this method would be called.

My assumption is that you want to check if `attr` is in one of the attributes that are not available on `DocumentError`, but are available on the successful result type, no?